pkg/controllers/secrets.go

```diff
@@ -74,6 +74,10 @@ func MapToEnvFormat(secrets map[string]string, wrapInQuotes bool) []string {
 	return env
 }
 
+func MapToDotNETFormat(secrets map[string]string) map[string]string {
```

Comment:
nit/nofix: This function isn't used in this commit so you could just add it in the next one. I think it's probably fine to leave as-is though 👍

Reply:
Will leave as is but will bear this in mind for next time.